app/controllers/events/view/export.js

```diff
@@ -0,0 +1,45 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
+export default Controller.extend({
+  eventExportStatus : '',
+  eventDownloadUrl  : '',
+  actions           : {
+    startGeneration() {
+      let payload = {
+        image    : this.get('model.exportData.dataImage'),
+        video    : this.get('model.exportData.dataVideo'),
+        audio    : this.get('model.exportData.dataAudio'),
+        document : this.get('model.exportData.dataDocument')
+      };
+      let exportUrl = `/events/${this.get('model').currentEventID}/export/json`;
+      let that = this;
+      this.get('loader')
+        .post(exportUrl, payload)
+        .then(data => {
+          let requestLoop = setInterval(function() {
+            that.get('loader')
+              .load(`${data.task_url.substring(3)}`)
```

Comment:
Don't use substrings for getting the path. Parse the URL properly and implicitly define which part of the URL to use. What does the value of `data.task_url` look like now ?